tensorflow/contrib/image/python/ops/image_ops.py

```diff
@@ -37,16 +37,18 @@
 ops.RegisterShape("ImageProjectiveTransform")(common_shapes.call_cpp_shape_fn)
 
 
-def rotate(images, angles, interpolation="NEAREST"):
+def rotate(images, angles, interpolation="NEAREST", name="rotate"):
```

Comment:
name=None and use the default second argument to name_scope below

Reply:
Done throughout.